app/src/main/java/test/FlickrResponseParserTest.java

```diff
@@ -0,0 +1,37 @@
+package test;
```

Comment:
nice though you should move it to appropriate package and use Roboelectric (to run tests on JVM, not Android)